src/Components/ContainerItem/ContainerItem.js

```diff
@@ -307,6 +325,35 @@ export default function ContainerItem({
 										</Heading>
 										{activePanel === 'objects' && (
 											<>
+												{objects && (objects.length !== 0 || (objects.length === 0 && search !== null)) && (
+													<Form.Control>
+														<Form.Input
+															placeholder="Search here..."
+															value={search ? search : ''}
+															size='small'
+															style={{ margin: '10px 0' }}
+															onChange={(e) => setSearch(e.target.value)}
+															onKeyDown={(e) => {
+																if(e.key === 'Enter') onGetObjects(containerItem.containerId, 0);
+															}}
+															onBlur={() => onGetObjects(containerItem.containerId, 0)}
+															disabled={isLoadingObjects}
+														/>
+														<div className="tooltip_block">
+															<img
+																className="input_icon"
+																src="/img/icons/info_circle.svg"
+																height={20}
+																width={20}
+																alt="info"
+															/>
+																<div className="tooltip">Flexible search allows you to use any attributes to search through the attribute=value formula, you can use parameters such as FilePath, FileName, etc.</div>
```

Comment:
Maybe we should specify here that we should write attributes by formula using commas, or is it obvious?

Reply:
Not really, we need an example, probably.